src/components/CustomModal/style.css

```diff
@@ -1,8 +1,8 @@
 /* Default and primary button colors differ within modals */
 .modal :global(.ant-btn-default) {
-    background-color: transparent;
+    background-color: var(--light-button-default-background);
     border-color: var(--light-theme-modal-btn-default-color);
-    color: var(--light-theme-modal-btn-default-color);
+    color: var(--light-button-default-color);
```

Comment:
these button styles are now redundant

Reply:
so we should import the light theme into the CustomModal component and delete modal button stylings entirely

Reply:
yeah, but not the other styling that's specific to modals